Resources/doc/index.rst

```diff
@@ -1,15 +1,167 @@
+
+..
+
+Now You must create new index for ElasticSearch:
+
+.. code-block:: bash
+
+    cd ..
+    app/console es:index:create
+
+..
+
+And import demo content for `ongr.dev <http://ongr.dev>`_ pages:
+
+.. code-block:: bash
+
+    app/console es:index:import --raw src/ONGR/DemoBundle/Resources/data/contents.json
+
+..
+
+Now you need to import data from magento to Your newly baked ONGR shop:
+
+.. code-block:: bash
+
+    app/console ongr:import:full magento.product
+    app/console ongr:import:full magento.category
 
-    ongr_magento:
-        shop: 1
 ..
 
-*'1' refers to store_id from which to import data*
+Step 7: Open your browser
+-------------------------
+
+Navigate your browser to `http://ongr.dev <http://ongr.dev/>`_
+
+Here You will find Your new shops front end.
+
+
+If You would visit `http://magento.ongr.dev/ <http://magento.ongr.dev/>`_  You would find original Magento e-shop with demo data.
 
-Also config database connection parameters in app/config/parameters.yml
+If You would like to check Magento administrators UI go to `http://magento.ongr.dev/admin <http://magento.ongr.dev/admin>`_
+
+    Username: admin
+
+    Password: admin123
+
+Step 7: Sync
+-------------------------
+
+
+If You wish to check how sync works You should make a change in Magento admin and run following commands:
```

Comment:
If you wish to check how sync works, you should make a change in Magento admin and run following commands: